meta/Meta.cpp

```diff
@@ -7446,18 +7446,31 @@ void Meta::meta_sai_on_fdb_event_single(
                 sai_attribute_t *list = data.attr;
                 uint32_t count = data.attr_count;
 
-                sai_attribute_t local[2]; // 2 for port id and type
-
-                if (count == 1)
+                //Add an attr of type if possible.
+                sai_attribute_t local[SAI_FDB_ENTRY_ATTR_END]; // 2 for port id and type
+                if (count <= SAI_FDB_ENTRY_ATTR_END)
```

Comment:
count in number of actual attributes, this comparison here with SAI_FDB_ENTRY_ATTR_END is invalid

Reply:
Just in case SAI will give an error return.